experiments/generic/pod_cpu_hog/pod_cpu_hog_k8s_job.yml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: pod-cpu-hog- 
+spec:
+  template:
+    metadata:
+      labels:
+        experiment: pod-cpu-hog
+    spec:
+      # Placeholder that is updated by the executor for automated runs
+      # Provide appropriate SA (with desired permissions) if executed manually
+      serviceAccountName: nginx 
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: litmuschaos/ansible-runner:ci  
+        imagePullPolicy: Always
+        env: 
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+
+          # provide application namespace
+          - name: APP_NAMESPACE
+            value: ''
+
+           # provide application labels
+          - name: APP_LABEL
+            value: ''
+
+           # provide application kind
+          - name: APP_KIND
+            value: ''
+
+          - name: TARGET_CONTAINER
+            value: ''
+
+          - name: CPU_CORES
+            value: '1'
+
+          - name: TOTAL_CHAOS_DURATION
+            value: '60000' # in ms
+
```

Comment:
provide lib - [supported platforms]
Also, we can add a comment on TARGET_CONTAINER provide target container name

Reply:
This experiment is not platform specific (app/container level cpu chaos..). However, added a not on the LIB and also target container